### PR TITLE
fix tps filter panic bug(pre-check the configuration of tps at the st…

### PR DESCRIPTION
**Pull request by Mulavar**
…art phase)

**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: config/method_config.go [hunk 1]
```diff
@@ -70,9 +76,34 @@ func initProviderMethodConfig(sc *ServiceConfig) error {
 }
 
 // check set default value and verify
-func (mc *MethodConfig) check() error {
-	if err := defaults.Set(mc); err != nil {
+func (m *MethodConfig) check() error {
+	qualifieldMethodName := m.InterfaceName + "#" + m.Name
+	if m.TpsLimitStrategy != "" {
+		_ = extension.GetTpsLimitStrategyCreator(m.TpsLimitStrategy)
```
Comment (flycash) [inline config/method_config.go:82]: why we call this? you don't use the return value.

Comment (flycash) [reply config/method_config.go:82]: I prefer that you add one more function to check if the creator exist. And then you log error here.

Comment (Mulavar) [reply config/method_config.go:82]: call this to check if we enable the corresponding strategy, maybe adding one more function is a better way to detect this, I'll fix it.

**Comment by LaurenceLiZhixin** [comment]
@Mulavar Please check this conflict.

**Review by AlexStocks — APPROVED** [review]

**Pull request merged**